src/search/mod.rs

```diff
@@ -0,0 +1,49 @@
+use crate::search::models::{SearchRequest, SearchMod};
+use meilisearch_sdk::client::Client;
+use meilisearch_sdk::search::Query;
+
+pub mod indexing;
+pub mod models;
+
+fn search(info: &SearchRequest) -> Vec<SearchMod> {
+    let client = Client::new("http://localhost:7700", "");
```

Comment:
Same as above, should use `dotenv` for url

Reply:
will do